source/facter/3.0/release_notes.md

```diff
@@ -5,6 +5,81 @@ title: "Facter 3.0: Release Notes"
 
 This page documents the history of the Facter 3.0 series. (Previous version: [Facter 2.4 release notes](../2.4/release_notes.html).)
 
+Facter 3.0.2
+-----
+
+Released July 15, 2015.
+
+Shipped in puppet-agent version: 1.2.2.
+
+Facter 3.0.2 fixes several regressions in Facter 3.0.0 and implements new features to support legacy workflows.
+
+### Feature: `--show-legacy` Flag Outputs Unstructured Facts
+
+Facter 3 outputs new, structured versions of many facts, such as `os`, and hides the equivalent unstructured legacy facts. In Facter 3.0.2, you can force Facter's output to include legacy facts by running it with the `--show-legacy` flag.
+
+- [FACT-1075](https://tickets.puppetlabs.com/browse/FACT-1075)
+
+### Feature: Improved Memory Usage on Launch
+
+Previous versions of Facter could consume more memory than was necessary on a `puppetserver` launch. Facter 3.0.2 significantly improves memory usage on launch, making `puppetserver` easier to start on low-spec systems.
```

Comment:
I'm not sure this is a feature, as prior to Facter 3.0 memory use wasn't an issue. The ticket is marked as a bug.